tcms/testplans/static/testplans/js/get.js

```diff
@@ -244,6 +244,40 @@ function toolbarEvents() {
         );
 
     });
+
+    $('.js-toolbar-sort-options li').click(function(ev) {
+        let sortBy = ev.target.innerText;
+
+        // update the sort button label
+        $('#sort-button')[0].innerHTML = sortBy + '<span class="caret"></span>';
+
+        sortBy = sortBy.toLowerCase();
+
+        let tcsParentElement = $('#testcases-list'),
+            visibleTCrows = $('.js-testcase-row:visible');
+
+        if (sortBy === 'status') {
+            sortBy = 'case_status';
+        }
```

Comment:
Unify both drop down lists (e.g. make them an includable HTML template) and use the data-filter-type attribute.